core/src/OpenMP/Kokkos_OpenMP_Parallel.hpp

```diff
@@ -199,19 +202,19 @@ class ParallelFor<FunctorType, Kokkos::MDRangePolicy<Traits...>,
   typename std::enable_if<!std::is_same<typename Policy::schedule_type::type,
                                         Kokkos::Dynamic>::value>::type
   execute_parallel() const {
-#pragma omp parallel for schedule(static KOKKOS_OPENMP_OPTIONAL_CHUNK_SIZE) \
-    num_threads(OpenMP::impl_thread_pool_size())
+#pragma omp parallel for schedule(static, 1) \
```

Comment:
Is that correct to just use `1` here?

Anyways, at least for the compilation to pass this change needs to be applied in line 193 as well.

Reply:
And if `1` is the correct value, it'd still be better to use named constant.